packages/database/src/Database.php

```diff
@@ -4,15 +4,17 @@
 
 namespace Tempest\Database;
 
+use Tempest\Database\Builder\QueryBuilders\BuildsQuery;
+
 interface Database
 {
-    public function execute(Query $query): void;
+    public function execute(BuildsQuery|Query $query): void;
```

Comment:
I've added support for `BuildsQuery` directly in these methods, so that you didn't have to call `->build()` explicitly when passing a query builder to the database.

Still TODO: rename the `BuildsQuery` interface to `QueryBuilder`, but then I need to find a new name for the `QueryBuilder` class.